distribution/kustomization.yaml

```diff
@@ -2,15 +2,15 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
-# non-kubeflow
-# - argocd-applications/metallb.yaml
+## Common
 - argocd-applications/argocd.yaml
-- argocd-applications/kubeflow-roles-namespaces.yaml
 - argocd-applications/cert-manager.yaml
-- argocd-applications/istio.yaml
+# - argocd-applications/istio.yaml
```

Comment:
Might be better to use the istio-operator and istio-upstream deployments by default to make it easier to customize and maintain the deployment.

Reply:
I don't really have a strong opinion about what goes into this kustomization.yaml at the moment. Since we are not really at a point where people are using it I suspect we'll be changing it more or less daily. We should maybe think about how we want to present people with options. For example, let them choose dex vs cognito or vanilla notebooks vs the one with monitoring built in. The easiest is probably to just document this directly in kustomize.yaml and make it clear that people should use one and comment out the other one.

As to istio though, with istio-upstream you mean directly from the istio repo instead of from kubeflow/manifests?

Reply:
Also, should we maybe place a big red "WORK IN PROGRESS" sign in the readme?